.github/workflows/ci.yml

```diff
@@ -55,17 +61,28 @@ jobs:
           path: ci-artifacts
 
   ci-tests:
-    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest]
+    runs-on: ${{ matrix.os }}
 
     steps:
       - uses: actions/checkout@v2
       - uses: actions-rs/toolchain@v1
       - name: ci-cargo-tests
         run: make ci-cargo-tests
+      - name: ci-tools-install-dependencies
+        run:   |
+               if [ "$RUNNER_OS" == "Linux" ]; then
```

Comment:
Wouldn't using `${{ matrix.os }}` (template expanded by the GitHub workflow runner) be clearer than depending on an additional environment variable? This would take values among {`ubuntu-latest`, `macos-latest`} in this case.

Reply:
Yeah, that looks better -- I'm definitely guilty of copying stackoverflow in the former :)